src/content-tags-drawer/tags-sidebar/TagsSidebarHeader.jsx

```diff
@@ -0,0 +1,43 @@
+import { Icon, Stack } from '@openedx/paragon';
```

Comment:
Missing type checking
```suggestion
// @ts-check
import { Icon, Stack } from '@openedx/paragon';
```

Reply:
Updated here https://github.com/openedx/frontend-app-course-authoring/pull/852/commits/1d505dad132b3194ae3326e4b59e01b69672e531